api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/MER_TX_CURRENT_2019.json

```diff
@@ -0,0 +1,635 @@
+{
+    "resourceType": "MeasureReport",
+    "id": "MER-TX-Indicators",
+    "status": "complete",
+    "type": "summary",
+    "measure": "https://ugisl.mets.or.ug/Measure/mer-tx-indicators",
+    "reporter": {
+        "reference": "Organisation/MK"
```

Comment:
@dbaluku what does this mean - Organisation/MK?

Reply:
Changed it to Organisation/UgandaEMR